### PR TITLE
Create journal-de-la-societe-des-oceanistes.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/85453/style-request-jso-journal-de-la-societe-des-oceanistes

### CODE REVIEW

File: journal-de-la-societe-des-oceanistes.csl [hunk 1]
```diff
@@ -0,0 +1,352 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <!-- Polyglot; journal publishes in English and French. -->
```
Comment (rmzelle) [inline journal-de-la-societe-des-oceanistes.csl:3]: @POBrien333, can you confirm the default-locale should be British English, and that this is not either a French or polyglot journal?

**Comment by adam3smith** [comment]
> can you confirm the default-locale should be British English, and that this is not either a French or polyglot journal?

the journal is polyglot English/French, but they do indeed use all English citations, even for French works in French articles, so the default locale is correct.

File: journal-de-la-societe-des-oceanistes.csl (hunk 1 above)
Comment (POBrien333) [reply journal-de-la-societe-des-oceanistes.csl]: as usual I forgot to take the locale out. thanks for spotting.

**Comment by POBrien333** [comment]
thanks for checking on this, Sebastian.

**Comment by rmzelle** [comment]
> the journal is polyglot English/French, but they do indeed use all English citations, even for French works in French articles, so the default locale is correct.

Great, thanks both. (I started adding the "Polyglot" XML comment as a reminder that a style should not carry a default-locale value, so in this case the comment should indeed be removed)

**Comment by rmzelle** [comment]
Thanks!

**Comment by POBrien333** [comment]
Looks like there is an issue with the styles being pushed over to zotero.org/styles.
Style is not appearing:
https://www.zotero.org/styles?q=journal-de-la-societe-des-oceanistes

**Comment by adam3smith** [comment]
It's there now. We're still fine-tuning the updated bot, so that'll occasionally happen

**Comment by POBrien333** [comment]
Thank you. :)

**Pull request merged**